app/src/main/java/com/example/parkingsystemjava/mvp/view/ParkingView.java

```diff
@@ -19,9 +20,12 @@ public ParkingView(Activity activity, ActivityMainBinding binding) {
 
     @Override
     public void showParkingsAvailable(int parkingsAvailable) {
-        Context context = getContext();
-        if (context != null) {
-            Toast.makeText(context, context.getString(R.string.toast_main_parking_size, parkingsAvailable), Toast.LENGTH_LONG).show();
-        }
+        FragmentManager fragmentManager = getFragmentManager();
+        DialogFragmentParkingLots editNameDialogFragment = DialogFragmentParkingLots.newInstance();
+        editNameDialogFragment.show(fragmentManager, "dialog_fragment_parking_lots");
+    }
+
+    public void showMessageOfParkingLots(int parkingsAvailable) {
+        Toast.makeText(getContext(), getContext().getString(R.string.toast_main_parking_size, parkingsAvailable), Toast.LENGTH_SHORT).show();
```

Comment:
please validate that the getContext is not null

Reply:
done